server/lib/src/constants/schema.rs

```diff
@@ -638,6 +638,27 @@ pub static ref SCHEMA_ATTR_LIMIT_SEARCH_MAX_FILTER_TEST_DL6: SchemaAttribute = S
     ..Default::default()
 };
 
+pub static ref SCHEMA_ATTR_LINKED_GROUP_DL6: SchemaAttribute = SchemaAttribute {
+    uuid: UUID_SCHEMA_ATTR_LINKED_GROUP,
+    name: Attribute::LinkedGroup.into(),
+    description: "A reference to the group linked to the entry".to_string(),
+
+    multivalue: false,
+    sync_allowed: true,
```

Comment:
Do you know what the sync_allowed attribute affects?

Reply:
Attribute or class can be imported from external scim?

Reply:
So I think the question is if there is a use case here where we would need to import a linked group? I think currently, the answer is no, so we should not set sync_allowed unless there is a compelling need.

Reply:
Changed to false.